packages/plus/package.json

```diff
@@ -30,9 +30,10 @@
   ],
   "sideEffects": false,
   "type": "module",
+  "types": "./dist/index.d.ts",
```

Comment:
why, it's should not be necessary !

Reply:
@philibea this is to be consistent between all package.json config among the repository